src/index.js

```diff
@@ -56,7 +56,8 @@ function createParser(parserFormat, transform, meta = {}) {
         options.printer = printers[original.astFormat]
       }
 
-      let ast = await original.parse(text, options)
+      // @ts-ignore: We pass three options in the case of plugins that are written to support Prettier 2 _and_ 3.
```

Comment:
Possible shorter version...

```suggestion
      // @ts-ignore: We pass three options in the case of plugins that support Prettier 2 _and_ 3.
```